src/stats/BasicRFunctions.java

```diff
@@ -51,7 +66,14 @@ public static double StudentKnowledgeEstAvg(KnowledgeEstimateMatrix loMatrix, St
         return actual;
     }
 
-    //Must be called at the start of every function that uses RCaller methods
+    /**
+     * Must be called at the start of every function that uses RCaller methods in
```

Comment:
This comment, coupled with the @pre tag used, seem to indicate side effect that calling this function in and of itself has some purpose,but this is not true, right?  I think the comment needs to be more like "returns an object hat must be used by anyone wanting to use RCaller methods..."